moviesurfer/data/download_data.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import os
+import zipfile
+
+import wget
+
+
+# Function to download data
+def download_data(
+    output_dir="moviesurfer/data",
+    url="http://files.grouplens.org/datasets/movielens/ml-25m.zip",
+):
+    """
+    Downloads the MovieLens 25M Dataset, extracts it, and deletes the zip file.
+
+    :param output_dir: str, the output directory to save the downloaded data
+    :return: None
+    """
+    logging.basicConfig(level=logging.DEBUG)
```

Comment:
I suggest removing this one. It can override existing logger config

Reply:
👍